migrations/sqls/20221007143236-jenkins-metrics-down.sql

```diff
@@ -0,0 +1 @@
+drop schema jenkins cascade;
```

Comment:
I believe you need to drop the tables before you drop the schema but I might be wrong. I guess if your test is passing then I am wrong.